tir/technologies/webapp_internal.py

```diff
@@ -8842,7 +8842,8 @@ def ClickCheckBox(self, label_box_name, position=1, double_click=False):
                         else:
                             self.send_action(action=self.click, element=label_box_element)
                             check_after_click = checked_status()
-                            success = check_after_click != check_before_click
+                            label_in_selenium = self.soup_to_selenium(label_box)
+                            success = check_after_click != check_before_click or label_in_selenium.get_attribute('checked') == 'true'
```

Comment:
```suggestion
                            success = check_after_click != check_before_click
```

Para manter a checagem sem precisar maiores atribuições sugiro mover a condição diretamente para o lambda ja existente acima


alterando a variavel na linha 8831 para:
```
                    checked_status =lambda: ((hasattr(self.get_checkbox_label(label_box_name, position), 'attrs') and
                                             'checked' in self.get_checkbox_label(label_box_name, position).attrs) or
                                            (self.soup_to_selenium(label_box).get_attribute('checked')))
```



Assim conseguimos manter a comparação apenas adicionando uma condição na função ja responsavel